### PR TITLE
Qt/Intro: Various fixes

### PR DESCRIPTION
**Pull request by luke-jr**

### CODE REVIEW

**Comment by MarcoFalke** [comment]
ACK on bumping the minimum, as this is part of the release process https://github.com/bitcoin/bitcoin/blob/master/doc/release-process.md#release-process

Would be great if someone could prepare a pull for all those bumps, so it could be done prior to the branch off.

Not sure if the other changes should go into 0.14.

**Comment by laanwj** [comment]
At least the changes that change translation messages cannot go into 0.14.

**Comment by luke-jr** [comment]
Stripped out the translation-touching stuff for now. I think this is good for 0.14.

**Comment by paveljanik** [comment]
ACK https://github.com/bitcoin/bitcoin/pull/9718/commits/5bb2a5ee0b4ec05722ce21afd65ba18240e7fbd1

File: src/qt/intro.cpp [hunk 1]
```diff
@@ -23,7 +23,7 @@
 
 static const uint64_t GB_BYTES = 1000000000LL;
 /* Minimum free space (in GB) needed for data directory */
-static const uint64_t BLOCK_CHAIN_SIZE = 80;
+static const uint64_t BLOCK_CHAIN_SIZE = 102;
```
Comment (TheBlueMatt) [inline src/qt/intro.cpp]: This seems low.

118G	/mnt/bitcoin/blocks

Comment (TheBlueMatt) [inline src/qt/intro.cpp]: Of which only

9.4G	/mnt/bitcoin/blocks/index

(-txindex).

Comment (luke-jr) [inline src/qt/intro.cpp]: We still store stale blocks forever (but those won't be downloaded in IBD).

**Comment by MarcoFalke** [comment]
BLOCK_CHAIN_SIZE is meant to include some buffer, taking into account
when the next bump of this constant is planned. Otherwise it would
already be outdated when the software is released. I think you can
just round up to the next full 10 GB or something.

File: src/qt/intro.cpp (hunk 1 above)
Comment (TheBlueMatt) [inline src/qt/intro.cpp]: This is on a node which started sync last week, so it shouldnt have many stale blocks.

**Comment by luke-jr** [comment]
Okay, bumped it to 120.

**Comment by sipa** [comment]
utACK a9baa6d742c0425af4fb048cbd85707143227e84

**Comment by achow101** [comment]
utACK a9baa6d742c0425af4fb048cbd85707143227e84

**Comment by TheBlueMatt** [comment]
I would hesitate to call this a bugfix that needs to happen last-hour for 0.14, as it seems rather inconsequential, but utACK a9baa6d742c0425af4fb048cbd85707143227e84.

**Comment by paveljanik** [comment]
reACK https://github.com/bitcoin/bitcoin/commit/a9baa6d742c0425af4fb048cbd85707143227e84

**Comment by sipa** [comment]
Agree it's mostly an improvement, but we should get the blockchain size estimate update in.

**Comment by MarcoFalke** [comment]
utACK a9baa6d

**Pull request merged**